MAUI/PDF-Viewer/Custom-Bookmark.md

```diff
@@ -110,6 +208,29 @@ if (bookmarkToRemove != null)
 }
 
 
+{% endhighlight %}
+{% endtabs %}
+
+### Tracking Removal of custom bookmark:
+
+In the `CollectionChanged` event handler, you can use the `NotifyCollectionChangedAction` enum to determine if the action is `Remove`. If it is, you iterate through all items in e.NewItems and log each entry to the debug console using **Debug.WriteLine**.
+
+{% tabs %}
+{% highlight c# %}
+
+// Event handler for changes in the bookmarks collection
+private void Bookmarks_CollectionChanged(object? sender, NotifyCollectionChangedEventArgs e)
+{
+    // Handle Removal of new bookmarks
+    else if (e.Action == NotifyCollectionChangedAction.Remove && e.OldItems != null)
+    {
+        foreach (Bookmark oldBookmark in e.OldItems)
```

Comment:
<img width="466" height="156" alt="image" src="https://github.com/user-attachments/assets/00b0dd99-3b8d-4b44-82a2-8b93af2c5ebf" />

This is old bookmark or removed bookmark. 

Instead of confusions, you may simply named as **bookmark**. if changing name to bookmark, address here also for consistency.
<img width="441" height="189" alt="image" src="https://github.com/user-attachments/assets/303ff225-5d14-401b-bcc8-c0f4bb2f3aed" />